front-end/src/components/SearchResult.vue

```diff
@@ -170,7 +176,7 @@ const downloadLabel = computed(
           <svg
             class="usa-icon"
             focusable="true"
-            aria-labelledby="green-check"
+            aria-label="green-check"
```

Comment:
No need for the hyphen, and the value here should be "Transaction can proceed." which is the value in the element with the ID of "green-check" previously.